src/editor/index.jsx

```diff
@@ -22,6 +22,8 @@ class FakeEditor extends React.Component {
   constructor() {
     super();
 
+    this.baseUrl = window.location.href.match(/(.*)\/editor/)[1]
+
```

Comment:
I think it better to create wrapper function like:
```ts
const baseUrl = window.location.href.match(/(.*)\/editor/)[1];
this.fetcher = (url, options) => {
   fetch(baseUrl + url, {
      credentials: 'include',
      ...options,
   }
}
```
And rename existing `fetcher` to `graphqlFetcher`
@JamesKyburz  What do you think?